src/material/checkbox/_checkbox-theme.scss

```diff
@@ -29,6 +30,8 @@
     }
 
     .mat-mdc-checkbox {
+      @include mdc-form-field-theme.theme(tokens-mdc-form-field.get-color-tokens($theme));
```

Comment:
Ideally we'd also include this only once in the `core-theme`. I tried it initially, but there were a lot of internal apps that customized the styles via the `checkbox-theme` instead of `core-theme`.